example/pom.xml

```diff
@@ -74,7 +74,17 @@ Also contains a sample Docker Compose file (see &quot;docker&quot; folder) for s
       <resource>
         <directory>${project.basedir}/docker</directory>
         <filtering>true</filtering>
-        <targetPath>${project.build.directory}/hono</targetPath>
+        <targetPath>${project.build.directory}/hono/example/docker</targetPath>
+      </resource>
+      <resource>
+        <directory>${project.basedir}/openshift</directory>
```

Comment:
@sophokles73 Sounds good to me
@ppatierno I was thinking about creating new assembly module, but in a way ti is an example deployment so it can stay like this for now. We can change the name of the file.